### ISSUE TITLE
Duplicate Device Group

### ISSUE THREAD

**Issue by coolwormgit** [Ylianst/MeshCentral#776]
Hello,
Every time i want to create a group i have to add users and setup the rights, its going to be more easy if you can add button to duplicate or copy from another group and just change the group name.
thanks

**Comment by MailYouLater** [comment]
Like using an existing group as a template to create the new group from.

**Comment by Ylianst** [comment]
This is a good request, I should have a "Clone Group" button next to the "Create Group" one. This said, I want to make sure it's well known that user groups are being worked on now and currently don't do anything. Hopefully over the coming two weeks it user groups should start working.

**Comment by Ylianst** [comment]
It's not often that I add a suggestion for a feature that is not enabled yet, but just published MeshCentral v0.4.6-v and added a way to duplicate an existing group. Let me know if this works. This version has the user groups user interface working pretty well, but user groups don't do anything yet.

![MC2-UserGroupClone](https://user-images.githubusercontent.com/1319013/71701796-a7a69f00-2d80-11ea-9a6c-f91fa4aceb69.png)

**Comment by MailYouLater** [comment]
... I thought this was for device groups ...

**Comment by Ylianst** [comment]
OOohhhhhh..... duplicate the DEVICE group. Well, I feel silly. Ok, that makes sense now.

**Comment by coolwormgit** [comment]
yes i was talking about device groups :) sorry i didnt point the group :)

**Comment by Ylianst** [comment]
No worries. I am deep into building user groups support right now and so, live in an alternative group universe. Once user groups are done, I will come back and do a lot more improvements like this one.
